File: loop-exercises/exercise6.py
width = int(input('Width? '))
height = int(input('Height? '))

row = ''
side = '*'
i = 0
l = 0

while i < width:
    row += '*'
    i += 1

while l < (width - 2):
    side += ' '
    l += 1

side += '*'

print(row)
for n in range(height-2):
    print(side)
print(row)

File: javascript-exercises/excercise-2/exercise6.js
function printBox(width, height){
    var x = "";
    i = 0;
    while (i < width){
        x += "*";
        i++;
    }
    console.log(x);
    var z = 0;
    a = "*";
    while (z < width - 2){
        a += " ";
        z++;
    }
    a += "*";
    b = 0;
    while (b < height - 2){
        console.log(a);
        b ++;
    }
    console.log(x);
    
}
printBox(7, 6);